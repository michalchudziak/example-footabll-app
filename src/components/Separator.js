/* @flow */
import { StyleSheet, Dimensions } from 'react-native';
import styled from 'styled-components/native';

const Separator = styled.View`
  backgroundColor: #000000;
  width: ${Dimensions.get('window').width}
  height: ${StyleSheet.hairlineWidth};
`;

export default Separator;
