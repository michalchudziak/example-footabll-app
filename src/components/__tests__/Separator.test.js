/* @flow */
import React from 'react';
import renderer from 'react-test-renderer';
import Separator from '../Separator';

describe('<Separator />', () => {
  it('should render correctly', () => {
    const tree = renderer.create(<Separator />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
