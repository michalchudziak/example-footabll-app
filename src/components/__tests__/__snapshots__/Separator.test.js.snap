// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Separator /> should render correctly 1`] = `
<View
  style={
    Array [
      Object {
        "backgroundColor": "#000000",
        "height": 0.5,
        "width": 750,
      },
      undefined,
    ]
  }
/>
`;
